crypto/keys/der/der_test.go

```diff
@@ -68,9 +68,9 @@ func TestFromProto(t *testing.T) {
 			t.Parallel()
 
 			signer, err := FromProto(test.keyProto)
-			if gotErr := err != nil; gotErr != test.wantErr {
```

Comment:
What's the warning about using `gotErr`?

Reply:
It's confusing the IntelliJ nilness inspector into thinking that values are being used without checking for errors.

`Error corresponding to variable 'signer' may not be nil`

I could suppress it with `/noinspection GoNilness` but didn't really want to have tool specific comments in the code.

Reply:
Maybe I'll just revert the error handling changes and turn off this inspection.

Reply:
Reverted.